package.json

```diff
@@ -73,34 +76,26 @@
 		},
 		"globals": [
 			"browser"
-		]
+		],
+		"parserOptions": {
+			"jsx": true
+		},
+		"settings": {
+			"import/resolver": {
+				"node": {
+					"extensions": [".js", ".jsx", ".ts", ".tsx"]
+				}
+			}
+		}
```

Comment:
Try https://github.com/xojs/eslint-config-xo-typescript It already comes with much of this config.

Reply:
And a lot of rules too, I think.

Reply:
Can you try applying it to this PR? 😏 I'm done

Reply:
Done: https://github.com/sindresorhus/refined-github/pull/1726/commits/94eba5ad6772b0e74b9c33f0f1b8926523fd594e The TODOs there can be done in follow-up PRs.

Reply:
Oops, just noticed this was about the linting config. I'll do that in a follow-up PR later on.